generator/testdata/google/example/library/v1/library_service_stub.gapic.cc.baseline

```diff
@@ -124,14 +126,208 @@ class DefaultLibraryServiceStub : public LibraryServiceStub {
     ::google::example::library::v1::GetBookRequest const& request,
     ::google::example::library::v1::Book* response) override {
     grpc::ClientContext grpc_ctx;
-    context->PrepareGrpcContext(&grpc_context);
+    context->PrepareGrpcContext(&grpc_ctx);
     return google::gax::GrpcStatusToGaxStatus(grpc_stub_->GetBigBook(&grpc_ctx, request, response));
   }
 
  private:
   std::unique_ptr<::google::example::library::v1::LibraryService::StubInterface> grpc_stub_;
 }; // DefaultLibraryServiceStub
 
+class RetryLibraryServiceStub : public LibraryServiceStub {
+ public:
+  RetryLibraryServiceStub(std::unique_ptr<LibraryServiceStub> stub_,
+                         gax::RetryPolicy const& retry_policy,
+                         gax::BackoffPolicy const& backoff_policy) :
+            next_stub_(std::move(stub)),
+            retry_policy_(retry_policy.clone()),
+            backoff_policy_(backoff_policy.clone()) {}
+
+  google::gax::Status
+  CreateBook(google::gax::CallContext& context,
+             ::google::example::library::v1::CreateBookRequest const& request,
+             ::google::example::library::v1::Book& response) {
+    auto retry_policy = (ctx.retry_policy_ ? ctx.retry_policy_ : retry_policy_)->clone();
```

Comment:
Consider a (private) member function to clone the retry policy:

```C++
  auto retry_policy = clone_retry_policy(ctx);
```

Reply:
Will do, and same for backoff_policy. I'm a Lisper, so I'll still be using ternaries :)